tools/SetupFlow/DevHome.SetupFlow/Views/ConfigurationFileView.xaml

```diff
@@ -37,6 +37,7 @@
     <UserControl.Resources>
         <ResourceDictionary>
             <converters:StringVisibilityConverter x:Key="StringVisibilityConverter"/>
+            <converters:EmptyObjectToObjectConverter x:Key="EmptyObjectToObjectConverter" EmptyValue="Visible" NotEmptyValue="Collapsed"/>

```

Comment:
EmptyObjectTo`Visibility`Converter.

Reply:
I don't think this converter class exists, but that's a good idea in general to have (however outside this PR scope)

Reply:
I am sorry.  I didn't mean to change the class name.  Change the Key to `EmptyObjectToVisibilityConverter`